en/docs/observe/api-manager-analytics/overview-of-api-analytics.md

```diff
@@ -1,221 +1,45 @@
-<td>Provides a short-term historical view about Latency, Traffic Volume, and Errors of APIs with drill-downs to narrow down errors and isolate the cause of the issues.</td>
-</tr>
-
-<tr>
-<th>
-Targeted Audience
-</th>
-<td>This dashboard is targeted for DevOps and SysAdmins who overlook the health and availability aspects of the APIs.
-</td>
-</tr>
-<tr>
-<th>
-Permission Needed
-</th>
-<td>Currently, the <code>internal/analytics</code> role is needed to view this dashboard. However, a different role can be assigned by editing role mappings as described in <a href="({{base_path}}/observe/api-manager-analytics/managing-dashboard-permissions">Managing Analytics Dashboard Permissions</a>.
-</td>
-</tr>
-<tr>
-
-<td colspan="2">For more details, see the <a href="{{base_path}}/observe/api-manager-analytics/viewing-api-statistics/monitoring-dashboard">Monitoring Dashboard</a> section.
-</td>
-</tr>
-</table>
-
-______
-<a name="monthly-api-usage-report"></a>
-<table>
-<tr>
-<th colspan="2"><b><a href="{{base_path}}/observe/api-manager-analytics/monthly-api-usage-report">Reports Dashboard</a></b></th>
-
-</tr>
-<tr>
-<th>Information Provided</th>
-<td>Users can download a PDF report of the API usage statistics for the selected month. Later this report can be used for billing purposes. 
-<div class="admonition note">
-<p class="admonition-title">Note</p>
-<p>You can customize the format and structure of this dashboard as explained in the <a href="{{base_path}}/observe/api-manager-analytics/monthly-api-usage-report/#customizing-the-report">Monthly API Usage Report</a> section.</p>
-
-</td>
-</tr>
-
-<tr>
-<th>
-Targeted Use case
-</th>
-<td>This can be used to accomplish the API monetization use cases.
-</td>
-</tr>
-<tr>
-<th>
-Permission Needed
-</th>
-<td>Currently, this is restricted to users with the <code>admin</code> role only.
-</td>
-</tr>
-<tr>
-
-<td colspan="2">For more details, see the <a href="{{base_path}}/observe/api-manager-analytics/monthly-api-usage-report">Monthly API Usage Report</a> section.
-</td>
-</tr>
-</table>
-
-## Customizing the Analytics Dashboards
-
-Apart from the latter mentioned default dashboards, the users can create custom dashboards based on their specific requirements as explained in **[Customizing Analytics Dashboards]({{base_path}}/observe/api-manager-analytics/customizing-analytics-dashboards)**.
+![APIM Analytics Overview Page]({{base_path}}/assets/img/observe/apim-analytics-overview-page.png)
+
+APIs are widely used in Business Integrations and are becoming a key part of driving an organization's business 
+strategy. Ensuring the proper functioning of the APIs, getting feedback on the performance and deriving business 
+insights from APIs are becoming equally important.
+
+With the API Manager 4.0.0 release API Analytics was completely revamped and offered as a cloud-based analytics solution
+ to provide users with an unparallel analytics experience addressing many customer concerns we received for earlier 
```

Comment:
```suggestion
 to provide users with an unparalleled analytics experience addressing many customer concerns we received for earlier 
```
Shall we avoid mention of customer concerns here? We don't normally introduce that sort of viewpoint in docs.